app-example-js/src/index.js

```diff
@@ -0,0 +1,44 @@
+
+// outsource dependencies
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import React, { useCallback, useEffect } from 'react';
+
+// local dependencies
+import './index.css';
+import store from './store';
+import logo from './logo.svg';
+import { controller } from './controller';
+import { useController } from 'redux-saga-controller';
+
+
+function App() {
+  const [
+    { data, disabled, initialized },
+    { initialize, getSelf },
```

Comment:
Maybe we should add isControllerConnected to example?

Reply:
In order to simplify example of usage i think No